### DISCUSSION TITLE
show content in different pages

### DISCUSSION THREAD

**Discussion by ncupertino** [tinymce/tinymce#9614] [General]
Hello

It is possible to show which page the document is on, that is, I need to identify which page my document is on before printing in PDF.

**Comment by lostkeys** [comment]
Hi @ncupertino! Unfortunately it's not possible in TinyMCE as HTML doesn't have the concept of running pages. There might be tools out there that can calculate this, but I'm not aware of any.

— Fredrik, Product Manager
